connect/runtime/src/test/java/org/apache/kafka/connect/runtime/standalone/StandaloneHerderTest.java

```diff
@@ -812,7 +861,7 @@ public void testPutConnectorConfig() throws Exception {
         FutureCallback<Herder.Created<ConnectorInfo>> reconfigureCallback = new FutureCallback<>();
         herder.putConnectorConfig(CONNECTOR_NAME, newConnConfig, true, reconfigureCallback);
         Herder.Created<ConnectorInfo> newConnectorInfo = reconfigureCallback.get(1000L, TimeUnit.SECONDS);
-        ConnectorInfo newConnInfo = new ConnectorInfo(CONNECTOR_NAME, newConnConfig, Arrays.asList(new ConnectorTaskId(CONNECTOR_NAME, 0)),
+        ConnectorInfo newConnInfo = new ConnectorInfo(CONNECTOR_NAME, newConnConfig, singletonList(new ConnectorTaskId(CONNECTOR_NAME, 0)),
```

Comment:
My IDE has been nagging me about this one too, for years. I've left it as-is (maybe not here but in other places) since it's not performance-critical and it's easier to modify in case we need to add other elements, or construct an empty list. I think we should revert this change.

Reply:
Haha yeah I only did this so that my IDE stopped nagging me 😄 

>it's easier to modify in case we need to add other elements, or construct an empty list

Fair point, I've reverted this.